Documentation/configuration/metrics.rst

```diff
@@ -41,7 +41,18 @@ Endpoint
 Datapath
 --------
 
-* ``datapath_errors_total``: Total number of errors occurred in datapath management, labeled by area, name and address family.
+* ``datapath_errors_total``: Total number of errors occurred in datapath
+  management, labeled by area, name and address family.
+* ``datapath_conntrack_gc_runs_total``: Number of times that the conntrack
+  garbage collector process was run. It contains a label status that describes
+  if it was successful or not.
+* ``datapath_conntrack_gc_key_fallbacks_total``: Number of times that the Key fallback
+  was invalid.
+* ``datapath_conntrack_gc_size``: The size of the conntrack table when the
```

Comment:
I think it's a "_count" now, rather than a "_size"? Since it includes deleted?

Reply:
`_count` should never be used except for histograms.
https://github.com/prometheus/prometheus/blob/master/util/promlint/promlint.go#L185